### PR TITLE
OSDOCS-1549, updated Operator installation docs for Dedicated

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://issues.redhat.com/browse/OSDOCS-1549

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@billmvt @wgordon17 PTAL. Thanks!

File: administering_a_cluster/dedicated-admin-role.adoc [hunk 1]
```diff
@@ -33,18 +33,11 @@ include::modules/dedicated-managing-quotas-and-limit-ranges.adoc[leveloffset=+1]
 [id="osd-installing-operators-from-operatorhub_{context}"]
 == Installing Operators from the OperatorHub
 
-{product-title} administrators can install Operators from a curated list
-provided by the OperatorHub. This makes the Operator available to all developers
-on your cluster to create Custom Resources and applications using that Operator.
+{product-title} administrators can install Operators from a curated list provided by the OperatorHub. This makes the Operator available to all developers on your cluster to create Custom Resources and applications using that Operator.
```
Comment (billmvt) [inline administering_a_cluster/dedicated-admin-role.adoc:36]: is `a curated list provided by` redundant? In other words, should we just say you can install from OperatorHub?

**Review by billmvt — COMMENTED** [review]
@ahardin-rh sorry I missed this earlier, LGTM except the one comment I left

File: administering_a_cluster/dedicated-admin-role.adoc (hunk 1 above)
Comment (ahardin-rh) [reply administering_a_cluster/dedicated-admin-role.adoc:36]: @billmvt That does sound better. Thanks for the review!

**Pull request merged**